RULES.md

```diff
@@ -331,3 +331,11 @@ Description: Check that S3 buckets have default encryption enabled.
 
 Trigger Type: ```Change Triggered```
 Required Parameter: ```None```
+
+### 38. Check the patch compliance of a set of instances based on their tag and SSM 
```

Comment:
Please remove this file from your PR. We are working on getting this file auto-generated.